pdm/_vendor/halo/_utils.py

```diff
@@ -4,10 +4,7 @@
 import codecs
 import platform
 import six
-try:
-    from shutil import get_terminal_size
-except ImportError:
-    from backports.shutil_get_terminal_size import get_terminal_size
+from shutil import get_terminal_size
```

Comment:
We can just skip the files under `_vendor/` directory and keep this untouched. It will affect our vendoring workflow